biggerquery/cli/config_utlis.py

```diff
@@ -0,0 +1,24 @@
+from importlib import import_module
+
+from biggerquery.configuration import BiggerQueryConfig
+
+
+def import_config(config_path):
+    object_name = config_path.split('.')[-1]
+    module_path = '.'.join(config_path.split('.')[:-1])
+    imported_module = import_module(module_path)
+
+    if not hasattr(imported_module, object_name):
+        raise BiggerQueryConfigNotFound("Object '%s' not found in module %s" % (object_name, module_path))
+    maybe_biggerquery_config = getattr(imported_module, object_name)
+    if not isinstance(maybe_biggerquery_config, BiggerQueryConfig):
```

Comment:
I think it's not a good idea. It makes harder for a user to create their own configuration.

For example, I personally use duck typing when I use google oauth libraries by creating fake token classes, and I'm happy as a user that they don't check types.

Reply:
done